src/api/components/source.js

```diff
@@ -435,6 +435,8 @@ module.exports = class ComponentSource extends EntitySource {
 
             if (view) { // it is a component
                 const nameMatch = dir.name;
+                const notes = matchFile(file => anymatch(`**/${nameMatch}.${source.get('files.notes')}.md`, source._getPath(file)) );
+                dirConfig.notes = notes && notes.isFile ? notes.getContentSync() : undefined;
```

Comment:
- should not assign `dirConfig.notes` here - the readme should already be discovered by the matcher when `dirDefaults` is assigned
- the isFile/getContentSync is redundant since in Component `get notes()` already does that.